src/textual/message.py

```diff
@@ -77,7 +77,7 @@ def __init_subclass__(
         cls.handler_name = f"on_{namespace}_{name}" if namespace else f"on_{name}"
 
     @property
-    def control(self) -> Widget | None:
+    def control(self) -> DOMNode | None:
```

Comment:
Possibly want to tweak the docstring to say node rather than widget now; or from the "end user" point of view does the intent of the property remain the same? (we want them to think about widgets)

Reply:
Yeah, might leave it as widget in the docs. It's more meaningful for the average user.